### PR TITLE
CI -- RPM dev build fixes:

### PR DESCRIPTION
**Pull request by mellery451**
skip signature checks and allow for all branches (not PRs).

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -281,42 +285,36 @@ try {
             builds['rpm'] = {
```
Comment (wilsonianb) [inline Jenkinsfile]: Can we do the rpm build for non-collaborators now (upon collaborator approval), or was there a reason besides the commit signature check to not?

Comment (mellery451) [reply Jenkinsfile]: seems like we can..I'll make the change.

File: Jenkinsfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env groovy
+#!/usr/bin/env grovy
```
[the remark this answers is not in the document]
Comment (mellery451) [reply Jenkinsfile]: derp - will force push that one back

**Comment by mellery451** [comment]
in 5a830b63e

**Pull request closed without merging**